lib/repl.js

```diff
@@ -566,6 +572,20 @@ REPLServer.prototype.createContext = function() {
     });
   });
 
+  Object.defineProperty(context, '_', {
+    configurable: true,
+    get: () => {
+      return this.last;
+    },
+    set: (value) => {
+      this.last = value;
+      if (!this.underscoreAssigned) {
+        this.underscoreAssigned = true;
+        this.outputStream.write('expression assignment to _ now disabled\n');
```

Comment:
I'd suggest capitalizing this print, it looks to be more in line with other things the REPL prints.